nni/retiarii/evaluator/pytorch/cgo/accelerator.py

```diff
@@ -90,17 +101,35 @@ def get_accelerator_connector(
         precision: int = 32,
         amp_backend: str = 'native',
         amp_level: str = 'O2',
-        plugins: Optional[Union[List[Union[Plugin, ClusterEnvironment, str]], Plugin, ClusterEnvironment, str]] = None):
+        plugins: Optional[Union[List[Union[Plugin, ClusterEnvironment, str]], Plugin, ClusterEnvironment, str]] = None,
+        **other_trainier_kwargs) -> AcceleratorConnector:
+    gpu_ids = Trainer()._parse_devices(gpus, auto_select_gpus, tpu_cores)
```

Comment:
`_parse_devices` is a staticmethod, so `Trainer._parse_devices` is enough.

Reply:
I notice `_parse_devices` of pytorch-lightning 1.4.2 is not staticmethod yet (https://github.com/PyTorchLightning/pytorch-lightning/blob/1.4.2/pytorch_lightning/trainer/trainer.py). The minor fix of addming staticmethod has not been merged.

Reply:
https://github.com/PyTorchLightning/pytorch-lightning/pull/8761